src/utils/SafeTransferLib.sol

```diff
@@ -12,15 +12,21 @@ library SafeTransferLib {
                              ETH OPERATIONS
     //////////////////////////////////////////////////////////////*/
 
+    error ETH();
+    error TRANSFER();
+    error TRANSFER_FROM();
+    error APPROVE();
```

Comment:
these errors are unclear, please retain the current messaging

Reply:
What do we wanna go with / add? 

Right now its just `require(success, "TRANSFER_FROM_FAILED");`

Which isnt really any different than saying "REVERT: TRANSFER_FROM"

Reply:
i still prefer wording it as failed, esp when theres no cost to a long error

Reply:
`TransferFromFailed` sgtm